gluster-block.toml

```diff
@@ -0,0 +1,5 @@
+GlusterBlockCLIPath = "/usr/bin/glusterd"
```

Comment:
This doesn't look right. Shouldn't this be `/usr/bin/gluster-block` ?

Reply:
updated